spec/models/tslr_claim_spec.rb

```diff
@@ -55,12 +55,21 @@
     end
 
     context "that has bank details" do
-      it "validates the length of account_number and account_sort_code" do
-        expect(TslrClaim.new(account_number: "123456789")).not_to be_valid
-        expect(TslrClaim.new(account_sort_code: "10111213")).not_to be_valid
-
-        expect(TslrClaim.new(account_number: "12345678")).to be_valid
-        expect(TslrClaim.new(account_sort_code: "101112")).to be_valid
+      it "validates the format of bank_account_number" do
+        expect(TslrClaim.new(bank_account_number: "ABC12 34 56 789")).not_to be_valid
+        expect(TslrClaim.new(bank_account_number: "12-34-56-78")).to be_valid
+      end
+      it "validates the length of bank_account_number" do
```

Comment:
I wonder if these two examples could be combined in to one `it` block? If the length of the account number is wrong it isn't in the correct format.

Reply:
👍